### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JanGut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,27 +2,31 @@ README
...
-- opencv-python
+Currently, the models of our neural networks are not available publicly.
+Due to their size, they are not included in the repository.
+Bit-Bots find them in the Mafiasi NextCloud `robocup-ai/log/`
+
+
+Vision Tools
+------------
+
+In the bitbots_vision_tools directory, special tools for debugging/introspection purposes are provided.
```
Comment (JanGut) [inline README.md]: add description for tools

File: bitbots_vision/package.xml [hunk 2]
```diff
@@ -6,21 +6,20 @@
...
     (bitbots_vision_cascade/config/visionparams.yaml). To be able to test
     new parameters, the debug_visionparams.yaml are provided. These are used
     when the vision is started with debug:=true as launchscript parameter.
     You do NOT want to enable DEBUG on the robot.
     The color calibration files are created with the wolves colorpicker and
-    a rosbag. The vision.py is located in bitbots_vision_cascade/scripts. It
-    defines, how the modules out of bitbots_vision_common are used. In
-    bitbots_vision_cascade/models the tensorflow classifier models are
+    a rosbag. The vision.py is located in the /src/bitbots_vision folder. It
+    defines, how the modules are used. In bitbots_vision/models the tensorflow classifier models are
```
Comment (NFiedler) [inline bitbots_vision/package.xml]: there is no classifier anymore.

Comment (JanGut) [reply bitbots_vision/package.xml]: resolved with 25f65b1.

File: README.md (hunk 1 above)
Comment (JanGut) [reply README.md]: Transfered to #42.

**Pull request merged**